collector/__init__.py

```diff
@@ -364,6 +367,21 @@ def _process_timing_windows(prog: Program, obs: Observation) -> List[Time]:
                 windows.extend([Time([begin + i * period, begin + i * period + duration]) for i in range(repeat)])
 
         return windows
+    
+    @staticmethod
+    def _calculate_proper_motion(target: Target, time: Time) -> SkyCoord:
+        """
+        Calculate the proper motion of a target.
+        """
+
+        pm_ra = target.pm_ra / Collector._MILLIARCSECS_PER_DEGREE
+        pm_dec = target.pm_dec / Collector._MILLIARCSECS_PER_DEGREE
+        
+        time_offsets = time - Collector._EPOCH2TIME[target.epoch]
+        new_ra = (target.ra + pm_ra * time_offsets.to(u.yr).value) * u.deg
+        new_dec = (target.dec + pm_dec * time_offsets.to(u.yr).value) * u.deg
+        return SkyCoord(new_ra, new_dec, frame='icrs', unit='deg')
```

Comment:
I'll make a separate Jira task to get `SkyCoord` out of here and convert the mini-model to use instead, in decimal degrees:
```
ra: npt.NDArray[float]
dec: npt.NDArray[float]
```

Reply:
The thing with getting rid of SkyCoord here is that the Collector uses `.separation` after returning the value. I think the angular distance method in horizons might be enough to replace but that is untested

Reply:
Ahhh... good call. I had forgotten about that. I see that you made a comment in SCHED-112 how we can get around this. Excellent. 👍